### ISSUE TITLE
Issues accessing LNURLp behind NGINX reverse proxy

### ISSUE THREAD

**Issue by kwilson62** [lnbits/lnurlp#10]
Hello, I have an nginx reverse proxy set up in front of LNBits on my Umbrel.

I can access LNBits with no problems both on local network and on clearnet.

However LNURLp has some problems.

For example, if I try using the LN address I made, I get this error:

2 validation errors for LnurlPayResponse callback URL host invalid, top level domain required (type=value_error.url.host) callback URL scheme not permitted (type=value_error.url.scheme; allowed_schemes={'https'})

I get the same error when trying to use the LNURL, the endpoints causing these issues are 

/lnurlp/{link_id}
/.well-known/lnurlp/{user}

**Comment by kwilson62** [comment]
I fixed this issue by editing the docker compose file and adding
`HOST: "mydomain.com"`
`FORWARDED_ALLOWED_IPS: "*"`

**Comment by ElucGeek** [comment]
I have the same issue with Lnbits on Umbrel. It used to work but it's not anymore.
I tried the 2 lines above but it's not solving. Sometimes it's preventing lnbits to start.

Where did you place the lines? 
Under service: app_proxy: environnement:?
Or under web: envrionnement:?
In the second one I have a:
LNBITS_HOST: "0.0.0.0"
I try to replace it by "my.domain.tld" but then it was not starting.

I tried the FORWARDED_ALLOW_IPS: "*" in both place but it's not solving anything.
